src/app/services/utility/master.forms.constants.ts

```diff
@@ -0,0 +1,1047 @@
+
+export class MasterForms {
+
+
+    public static CREATE_USER_FORM_TEMPLATE: any = {
+        "title": "Create User",
+        "components": [{
+            "label": "First Name",
+            "labelPosition": "top",
+            "labelWidth": "",
+            "labelMargin": "10",
+            "placeholder": "",
+            "description": "",
+            "tooltip": "",
+            "prefix": "",
+            "suffix": "",
+            "widget": {
```

Comment:
we can remove many unused properties from this. we only need few of them to render form correctly